pkg/util/rpcutil/log_call.go

```diff
@@ -0,0 +1,36 @@
+package rpcutil
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// LogCall is used to log an RPC call from the rpc.Server
+func LogCall(log logrus.FieldLogger, method string, in interface{}) func(out interface{}, err *error) {
+
+	// Just in case log is not set.
```

Comment:
Are you planning to fix a possible race condition or leave it as it is?

Reply:
Actually there wouldn't be any race conditions. I forgot to remove this comment.